src/clj/webchange/assets/handler.clj

```diff
@@ -73,7 +72,9 @@
           {:url relative-path
            :type type
            :size (normalize-size size)}
-          params)))))
+          params)))
+    (core/store-asset-hash! path)
```

Comment:
The last line in the function body is what returned as a result. 
in this case, it should be a map with url, type and params.

You need to call (core/store-asset-hash! path) earlier.